src/pybind/mgr/dashboard/frontend/src/app/shared/components/details-card/details-card.component.scss

```diff
@@ -0,0 +1,24 @@
+@use '@carbon/layout';
+@use './src/styles/vendor/variables' as vv;
+
+.details-card {
+  background-color: vv.$white;
+  box-shadow: 0 layout.$spacing-01 layout.$spacing-03 rgba(0, 0, 0, 8%);
```

Comment:
couldn't find a proper carbon token for this...

Reply:
I don't think shadow should be added as per: https://carbondesignsystem.com/components/tile/usage/#when-not-to-use